playbooks/provisioning/openstack/post-install.yml

```diff
@@ -0,0 +1,57 @@
+---
+- hosts: OSEv3
+  gather_facts: False
+  become: True
+  tasks:
+    - name: Save iptables rules to a backup file
+      when: openshift_use_flannel|default(False)|bool
+      shell: iptables-save > /etc/sysconfig/iptables.orig-$(date +%Y%m%d%H%M%S)
+
+# Enable iptables service on app nodes to persist custom rules (flannel SDN)
+# FIXME(bogdando) w/a https://bugzilla.redhat.com/show_bug.cgi?id=1490820
+- hosts: app
+  gather_facts: False
```

Comment:
`gather_facts` here should be set to `True`.

The playbook is run manually after openshift-ansible and if the facts cache gets cleared in the meantime, `os_firewall` will fail with:

```
TASK [os_firewall : Install iptables packages] ************************************************************************************************************************
Wednesday 18 October 2017  10:25:46 +0000 (0:00:00.034)       0:00:01.264 ***** 
failed: [app-node-0.openshift.example.com] (item=iptables) => {"failed": true, "item": "iptables", "msg": "Could not detect which package manager to use. Try gathering
 facts or setting the \"use\" option."}
failed: [app-node-0.openshift.example.com] (item=iptables-services) => {"failed": true, "item": "iptables-services", "msg": "Could not detect which package manager to 
use. Try gathering facts or setting the \"use\" option."}
```

Reply:
nice catch, will follow-up